Makefile

```diff
@@ -61,3 +61,20 @@ binaries: ${PREFIX}/bin/registry ${PREFIX}/bin/registry-api-descriptor-template
 clean:
 	@echo "+ $@"
 	@rm -rf "${PREFIX}/bin/registry" "${PREFIX}/bin/registry-api-descriptor-template"
+
+# import the existing docs build cmds from docker/docker
+DOCS_MOUNT := $(if $(DOCSDIR),-v $(CURDIR)/$(DOCSDIR):/$(DOCSDIR))
+DOCSPORT := 8000
+GIT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD 2>/dev/null)
```

Comment:
The string `v2.0.0-alpha.2-81-g8b3a5e4` should checkout an exact revision if the tags are defined. The branch name is basically meaningless. This may not work in `docker/docker` since they use a different merge strategy for releases.